### ISSUE TITLE
Rescue cardiovascular disease as TA in EFO slim

### ISSUE THREAD

**Issue by d0choa** [opentargets/platform#892]
Based on recent discussion on ACE2 invalid evidence (#890) and an analysis on the impact of missing evidence for CVD, we decided it's sensible to rescue the cardiovascular disease term from EFO. 

A ticket has been opened EBISPOT/efo#740 to action this task.

Once the new EFO OT profile is available, we need to double check it contains the right TAs and structure.

No further action is expected in the production pipeline.

**Comment by AsierGonzalez** [comment]
These are the number of invalid evidence strings we have per data source because they are annotated with `EFO_0000319`:

```
 640 "chembl"
28504 "europepmc"
3888 "ot_genetics_portal"
 377 "phewas_catalog"
Total: 33,409
```

**Comment by d0choa** [comment]
This change was implemented in EFO 3.16. It didn't cause any unindented issues in pipeline or FE (QA version). A reduction on the number of invalid strings has been confirmed.
